diagnostic.rb

```diff
@@ -3,14 +3,31 @@
 
 # question 2:
 
+caren = Licensee.create surname: 'Bishop',
+                        given_name: 'Caren',
+                        gender: 'f',
+                        height: 63,
+                        weight: 132,
+                        born_on: '1943-09-26',
+                        eye_color: 'Brown',
+                        hair_color: 'Black';
+
+
 # question 3:
 Licensee.transaction do
   CSV.foreach 'data/licensees.csv', headers: true do |row|
+    licensee = row.to_hash
+    Licensee.create! licensee unless Licensee.exists? licensee
   end
 end
 
 # question 4:
 
+licensees = Licensee.where(eye_color: 'Hazel').where(hair_color: 'Brown').or(hair_color: 'Black')
```

Comment:
`.where` is right, and you're right that you can chaing multiple `.where` calls. Unfortunately, that `.or` method doesn't exist .... yet. It's a feature that will be added to ActiveRecord in Rails 5 when it gets released (which should be sometime over the next few months).